test/tools/ossfuzz/yulProto_diff_ossfuzz.cpp

```diff
@@ -53,6 +53,8 @@ DEFINE_PROTO_FUZZER(Program const& _input)
 	if (yul_source.size() > 1200)
 		return;
 
+	YulStringRepository::reset();
```

Comment:
iiuc, test harnesses that invoke the Assembly stack parser need to make an explicit call to the reset function. Is that right?

Reply:
Yes, that is true.

Reply:
okay. Could you please add the reset in the following locations as well?

https://github.com/ethereum/solidity/blob/195a7ff61a0496c9a92ef0c97fc4d9c695a99c67/test/tools/ossfuzz/strictasm_assembly_ossfuzz.cpp#L29

https://github.com/ethereum/solidity/blob/195a7ff61a0496c9a92ef0c97fc4d9c695a99c67/test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp#L57

https://github.com/ethereum/solidity/blob/195a7ff61a0496c9a92ef0c97fc4d9c695a99c67/test/tools/ossfuzz/strictasm_opt_ossfuzz.cpp#L28

https://github.com/ethereum/solidity/blob/195a7ff61a0496c9a92ef0c97fc4d9c695a99c67/test/tools/ossfuzz/yulProtoFuzzer.cpp#L48